src/material/checkbox/checkbox.scss

```diff
@@ -517,6 +518,12 @@ $_mat-checkbox-mark-stroke-size: 2 / 15 * checkbox-common.$size !default;
 .mat-checkbox-input {
   // Move the input to the bottom and in the middle.
   // Visual improvement to properly show browser popups when being required.
-  bottom: 0;
-  left: 50%;
+  $offset: checkbox-common.$size / 2;
```

Comment:
Use `math.div` instead of the slash operator because it's going to change eventually
https://sass-lang.com/documentation/breaking-changes/slash-div

Reply:
I ended up reverting it, because the `math.div` function doesn't exist yet, even though we're on the latest version of Sass. It's not in the docs either: https://sass-lang.com/documentation/modules/math

Reply:
Huh, I thought they made that change already